xbmc/platform/win32/WIN32Util.cpp

```diff
@@ -1514,3 +1515,63 @@ VideoDriverInfo CWIN32Util::GetVideoDriverInfo(const UINT vendorId, const std::w
 
   return info;
 }
+
+std::wstring CWIN32Util::GetDisplayFriendlyName(const std::wstring& gdiDeviceName)
+{
+#ifdef TARGET_WINDOWS_STORE
+  // Not supported - not implemented yet
+  return std::wstring();
+#else
+
+  uint32_t pathCount{};
+  uint32_t modeCount{};
```

Comment:
I think latest guide recommends use explicit for primitive types i.e `{false}` `{0}` and reserves `{}` for structs, etc. but i'm a bite lost here.... Anyway is not much important.

Reply:
I went with what's in the current guidelines 5.4, but can change it either way.

Reply:
You are right that it is not updated in guidelines yet. Its your choice.